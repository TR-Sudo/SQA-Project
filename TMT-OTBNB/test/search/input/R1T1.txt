login
user01
search
Oshawa89
Oshawa
25
5
logout
quit