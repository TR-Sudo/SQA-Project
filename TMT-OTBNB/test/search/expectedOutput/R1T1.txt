Enter transaction: Enter username: You have been logged in
Enter transaction: Enter city: Error: City name must be less than 25 characters and can only contain A-z, spaces, and - or strictly be a *
Enter city: Enter rental price (0-999.99): Enter number of rooms (0-9): 
Search Results:

Rental ID: aaaabbb2
City: Oshawa
Number of bedrooms: 5
Rental price per night: 20.0

End of search results

Enter transaction: You have been logged out
Enter transaction: 